css/styles.css

```diff
@@ -125,6 +125,144 @@
     margin: 0;
 }
 
-.left-part{
+.left-part {
     position: relative;
+}
+
+
+@media only screen and (max-width: 860px) {
+
+    .categories {
+        margin: 1rem;
+        font-size: 80%;
+        font-weight: 600;
+        grid-gap: 1rem;
+    }
+
+    .category-card>p {
+        padding: 0.1rem;
+    }
+
+    .banner {
+        margin: 1rem;
+        letter-spacing: 1px;
+    }
+
+    .featured {
+        margin: 1rem;
+        grid-gap: 0.5rem;
+    }
+
+    .feat-img img {
+        height: 3rem;
+        width: 5rem;
+    }
+
+    .new-item-card {
+        margin: 1rem 1rem;
+    }
+
+
+    .banner-content h1 {
+        font-size: 100%;
+    }
+
+    .banner-content p {
+        font-size: 100%;
+    }
+
+    .banner-content button {
+        font-size: 100%;
+    }
+
+    .new-item-card .card-horizontal {
+        flex-direction: column;
+    }
+
+}
+
+
+@media only screen and (max-width: 540px) {
+
+    .categories {
+        margin: 0.5rem;
+        font-size: 60%;
+        grid-gap: 0.5rem;
+    }
+
+    .banner {
+        margin: 0.5rem 0.5rem;
+    }
+
+    .banner-content button {
+        font-size: 80%;
+    }
+
+    .banner-content{
+        left:5%;
+    }
+
+    .featured {
+        margin: 0.5rem 0.5rem;
```

Comment:
`margin:0.5rem;`

Reply:
Thankyou for pointing it out Mahesh, will do that